src/components/modals/SwapKYCModal.js

```diff
@@ -40,7 +40,9 @@ class SwapKYCModal extends Component<Props, State> {
     }
   }
   getToken = async (code: string) => {
-    const text = ENV.SHAPESHIFT_CLIENT_ID + ':' + ENV.SHAPESHIFT_SECRET
+    const id = ENV.SHAPESHIFT_CLIENT_ID ? ENV.SHAPESHIFT_CLIENT_ID : '3a49c306-8c52-42a2-b7cf-bda4e4aa6d7d'
+    const secret = ENV.SHAPESHIFT_SECRET ? ENV.SHAPESHIFT_SECRET : 'CWmm11jKoayEGPptfLzkyrrmyVHAG1skzRQuKJZYBrhy'
```

Comment:
You can use `||` in cases like these, so `const id = ENV.SHAPESHIFT_CLIENT_ID || '3a49c306-8c52-42a2-b7cf-bda4e4aa6d7d'`. The ternary operator is for when you want a more specific test than truthy / falsy like `!= null`.